internal/cucumber/config_test.go

```diff
@@ -0,0 +1,167 @@
+package cucumber
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/saucelabs/saucectl/internal/saucereport"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCucumber_FilterFailedTests(t *testing.T) {
+	testcases := []struct {
+		name      string
+		suiteName string
+		report    saucereport.SauceReport
+		project   *Project
+		expResult []string
+		expErr    error
+	}{
+		{
+			name:      "suite exists and failed tests exist",
```

Comment:
Nitpick: Admittedly, I'm not great at naming my tests 😅 I think it'd be even more beneficial to the developer if the test described its intent. "suite exists and failed tests exist" describes the input, but not the intent.